src/onboarding/Onboarding.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="app">
+    <div class="row">
+      <div class="col-3 screen-left">
+        <div class="left-content">
+          <p :class="{ active: step === 2 }" @click="toStep(2)">Account</p>
+          <p :class="{ active: step === 3 }" @click="toStep(3)">Permission</p>
+          <p :class="{ active: step === 4 }" @click="toStep(4)">Done</p>
+        </div>
+      </div>
+      <div class="col-6 screen-right">
+        <div class="right-content">
+          <div class="header">
+            <img class="sl-logo" src="/images/horizontal-logo.svg" />
```

Comment:
please make sure to add `alt` tag for all `img`, this helps users who can't see or use a screen reader :)